lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1452,4 +1452,10 @@ function dosomething_user_views_pre_render(&$view) {
   if ($view->name == 'user_search' && count($view->result) == 1) {
     drupal_goto('user/' . $view->result[0]->uid);
   }
+
+  if ($view->name == 'content_search') {
+    // Remove grouping labels
+    $table_style = 'table caption { display: none }';
+    drupal_add_css($table_style, $option['type'] = inline);
```

Comment:
Should this be `drupal_add_css($table_style, ['type' => 'inline']);`? The syntax you used looks a little funky.